.gitignore

```diff
@@ -6,6 +6,12 @@ _deps/
 
 # .c file generated by Cython:
 cython/sharp.c
+cython/cmisc.c
+cython/distances.c
+cython/srcsim.c
+
+# Fortran files generated by f2py
+fortran/*.f90
```

Comment:
There are F90 files not generated by f2py in fortran/ (and in the git tree currently) though.  Will such files need to be manually added in the future? (I wasn't sure if gitignore is case sensitive)

Reply:
My bad, I have changed that to only those generated by f2py now.